test/MetaTx.js

```diff
@@ -0,0 +1,497 @@
+const lightwallet = require('eth-lightwallet')
```

Comment:
Support for metaTxs probably needs to be added to eth-signer anyway, since that is what is used in the mobile app.